in.co.nandakumar.vara.yml

```diff
@@ -21,4 +21,4 @@ modules:
     sources:
       - type: archive
         url: https://nandakumar.co.in/software/vara/downloads/0.1/vara-0.1.240602.tar.gz
-        sha256: e40909b7d62de08ec6b891ad9b9bde515caf3a9345ae081ecc6362da97f64b0f
+        sha256: fac61f516ab6cda010cff8393d276ce5307c3378772ce7ab10cb676d77b57573
```

Comment:
generating a new tarball without changing the name is a like retagging: a BAD practice.